tests/integration/conftest.py

```diff
@@ -57,8 +57,9 @@ async def app(ops_test: OpsTest, app_name: str):
 
     # Add required relations
     await ops_test.model.add_relation(hello_kubecon_app_name, app_name)
-    await ops_test.model.wait_for_idle(timeout=10 * 60)
-
+    await ops_test.model.wait_for_idle(
+        timeout=10 * 60, status=ActiveStatus.name  # type: ignore[has-type]
```

Comment:
Could this go to the top of the file as a constant so we don't have to endlessly repeat the `type: ignore[has-type]`?